src/resources/InsightPanelConfigs/InsightPanelConfigInterfaces.ts

```diff
@@ -1,7 +1,9 @@
+import {Paginated} from '../BaseInterfaces';
+
 /**
  * Defines the options to list Insight Panel configurations.
  */
-export interface InsightPanelConfigListOptions {
+export interface InsightPanelConfigListOptions extends Paginated {
```

Comment:
Just to be sure: I've noticed that a few interfaces extends `Paginated` now, but there was no `page` and `perPage` parameters set prior to that. Is it because this information was either implicit or not enforced before?

Reply:
I think you might have not checked the whole interfaces.
I did not remove the definition of `pages` and `perPages` in an interface if the definition was documented.

For example here, well, check just 6 line below :P